packages/react-components/src/components/knowledge-graph/KnowledgeGraphPanel.tsx

```diff
@@ -208,31 +213,32 @@ export const KnowledgeGraphContainer: React.FC<KnowledgeGraphInterface> = ({
   return (
     <Container header={<Header variant='h3'>Knowledge Graph</Header>}>
       <div style={{ width: '100%', height: '100%' }}>
-        <SpaceBetween direction='vertical' size='s'>
-          <SpaceBetween direction='horizontal' size='s'>
-            <Input
-              type='text'
-              value={searchTerm}
-              onChange={(e) => {
-                setSearchTerm(e.detail.value);
-              }}
-            ></Input>
-            <Button onClick={onSearchClicked} data-testid='search-button'>
-              <FormattedMessage
-                id='KnowledgeGraphPanel.button.search'
-                defaultMessage='Search'
-                description='Search button text'
-              />
-            </Button>
-          </SpaceBetween>
-          {/* inline styling here for testing only this will be fixed in the next PR */}
+        <SpaceBetween direction='vertical' size='xl'>
+          <div style={{ minWidth: '500px' }}>
```

Comment:
Why `500px`? This would create issues in the future with responsive support. Suggest using a min of `400px` if possible.

Reply:
Smaller phones are usually max-width around `400px`

Reply:
This is to align with the canvas minWidth which is 500px (after experimentation this is the best minWidth for the graph canvas).